### ISSUE TITLE
Another question, any easy way to configure this to work with siri

### ISSUE THREAD

**Issue by dmoos777** [bwssytems/ha-bridge#106]
Another question, any easy way to configure this to work with siri.

**Comment by emiliosic** [comment]
@dmoos777 Check https://github.com/nfarina/homebridge

**Comment by bwssytems** [comment]
Also, any application that can control a Hue Bridge can control this ha-bridge as well. Google is your friend.

**Comment by dmoos777** [comment]
I must have something not quite right on the bridge then.  Works fine with Amazon Echo, but I can't get to play nice with anything else.  Do I need to have a real account with phillips hue?  Also not sure how to how the user id and password fit into this.  I need to find time to do more research to further narrow things down.  Also do I need to have SSL or certificates or something line that set up.

**Comment by bwssytems** [comment]
If you use this bridge with anything else, you have to run it on port 80 as it seems all applications built to talk to the hue defaulted to that port as that is what the real Hue bridge uses.

**Comment by dmoos777** [comment]
Ah that could explain a few things, first thing I did was change the port.  I also actually run 3 different copies of the bridge because I have about 130 devices now.

**Comment by bwssytems** [comment]
Is it working for you now?

**Comment by dmoos777** [comment]
I should have time to test it tomorrow.  I will let you know what I find.  Thanks Again!!!

**Comment by dmoos777** [comment]
Hmmn, running into issues tying to move the HA-Bridge to port 80.  I am using a raspberry pi3.  I followed http://coreyswrite.com/tips-tricks/amazon-echo-x10-home-control/ to allow me to control all of my X10 devices, which installs Apache and runs on port 80 by default.  I modified apache and moved that to port 8085, but when I tried to reconfigure the HA-Bridge for port 80 it dies with ignite failed
java.net.SocketException: Permission denied.   Any ideas?

Here is the complete output:

016-05-07 13:51:27,285 [main] INFO  com.bwssystems.HABridge.HABridge - HA Bridge (v2.0.3) starting....
2016-05-07 13:51:27,353 [main] INFO  com.bwssystems.HABridge.BridgeSettings - reading from config file: data/habridge.config
2016-05-07 13:51:27,664 [main] INFO  com.bwssystems.HABridge.HABridge - HA Bridge (v2.0.3) initializing....
2016-05-07 13:51:27,732 [main] INFO  com.bwssystems.HABridge.SystemControl - System control service started....
2016-05-07 13:51:28.304:INFO::Thread-0: Logging initialized @2851ms
2016-05-07 13:51:28,321 [main] INFO  com.bwssystems.HABridge.devicemanagmeent.DeviceResource - HABridge device management service started....
2016-05-07 13:51:28,750 [Thread-0] INFO  spark.webserver.JettySparkServer - == Spark has ignited ...
2016-05-07 13:51:28,760 [Thread-0] INFO  spark.webserver.JettySparkServer - >> Listening on 0.0.0.0:80
2016-05-07 13:51:28.779:INFO:oejs.Server:Thread-0: jetty-9.3.z-SNAPSHOT
2016-05-07 13:51:28,958 [Thread-0] ERROR spark.webserver.JettySparkServer - ignite failed
java.net.SocketException: Permission denied
        at sun.nio.ch.Net.bind0(Native Method)
        at sun.nio.ch.Net.bind(Net.java:433)
        at sun.nio.ch.Net.bind(Net.java:425)
        at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:223)
        at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:74)
        at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:326)
        at org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80)
        at org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:244)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
        at org.eclipse.jetty.server.Server.doStart(Server.java:384)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
        at spark.webserver.JettySparkServer.ignite(JettySparkServer.java:131)
        at spark.SparkInstance.lambda$init$0(SparkInstance.java:341)
        at java.lang.Thread.run(Thread.java:745)

**Comment by audiofreak9** [comment]
@dmoos777 (Corey here aka @audiofreak9), I recently [updated my post](http://coreyswrite.com/tips-tricks/amazon-echo-x10-home-control-updated/) since the ha-bridge can now make script calls.  If you do not use the Bridge Control index.php file you can follow my updated guide, it does not require Apache & PHP to work.

**Comment by dmoos777** [comment]
Thanks Corey.   I will look into that.  Figured out what was going on with the ha bridge.  It was a permissions issue.  I was running the ha bridge under a user ID, found out for some reason I can't run the ha-bridge on port 80 as a normal user.  Runs fine on other ports, and I was able to start it on port 80 from root. So I will need to dig a little deeper to figure out what the issue is.  Then I will decide if I want to have anything else running on the system that would require apache.  Meanwhile I was able to start homebridge and am playing with the config now.  Thanks to Corey, Emilio, and of course to BWS Systems
